app/cafe.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"{visitor['name']} is not vaccinated.")
+
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date < datetime.date.today():
+            raise OutdatedVaccineError(
+                f"{visitor['name']}'s vaccine is outdated."
+            )
+
+        if not visitor.get("wearing_a_mask", False):
+            raise NotWearingMaskError(
+                f"{visitor['name']} is not wearing a mask."
+            )
```

Comment:
Ensure that a string is returned when raising the `NotWearingMaskError` exception. This is required by the task description to ensure consistency in return types.